src/Eurofurence.App.Server.Web/Controllers/AnnouncementsController.cs

```diff
@@ -59,23 +69,45 @@ public async Task<ActionResult> DeleteAnnouncementAsync([FromRoute] Guid id)
         }
```

Comment:
Maybe a 204 status should be returned here instead of a 200, when no body is returned